src/main/java/com/notification_demo/service/NotificationServiceImpl.java

```diff
@@ -0,0 +1,21 @@
+package com.notification_demo.service;
+
+import com.notification_demo.entities.Notification;
+import com.notification_demo.repo.NotificationRepo;
+import org.aspectj.weaver.ast.Not;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+
```

Comment:
remove line. Always format the code by pressing ctrl+alt+L and ctrl+alt+o in intelliJ. Do this for all the files.